src/components/AddMeeting.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { useNavigate } from "react-router-dom"
+
+export default function AddMeeting() {
+  const navigate = useNavigate()
+  const params = useParams()
+  const [date, setDate] = useState("")
+  const [time, setTime] = useState("")
+  const [location, setLocations] = useState("")
+  const [contactInfo, setContactInfo] = useState([])
+
+  function handleDate(event) {
+    const inputValue = event.target.value
+    setDate(inputValue)
+  }
+
+  function handleTime(event) {
+    const inputValue = event.target.value
+    setTime(inputValue)
+  }
+
+  function handleLocation(event) {
+    const inputValue = event.target.value
+    setLocations(inputValue)
+  }
+
+  useEffect(() => {
```

Comment:
I like how you have applied the meetings to the contact object. I did it another way and it made it more convoluted. This is really nice and clear